Sources/Property.swift

```diff
@@ -371,6 +371,25 @@ extension PropertyProtocol {
 	}
 }
 
+extension PropertyProtocol where Value == Bool {
+	/// Returns a property with the oposite value
```

Comment:
Please follow the [documentation guidelines](https://github.com/ReactiveCocoa/ReactiveSwift/blob/master/Documentation/DocumentingCode.md). 😸 

```swift
 	/// Start the producer, then block, waiting for the last value.
 	///
 	/// When a single value or error is sent, the returned `Result` will
 	/// represent those cases. However, when no values are sent, `nil` will be
 	/// returned.
 	///
 	/// - returns: Result when single `value` or `failed` event is received.
 	///            `nil` when no events are received.
 	public func last() -> Result<Value, Error>? {
 		return take(last: 1).single()
 	}
```